backend/geneaprove/importers/gedcomimport.py

```diff
@@ -1431,7 +1417,7 @@ def _process_OBJE(self, obje, unless_in=set(), source=None,
                 file=f[0],
                 mime_type=f[1],
                 comments=comments +
-                   ("\nFormat: %s" % f[2] if f[2] else ""))
+                   (f"\nFormat: {f[2] if f[2] else ''}"))
```

Comment:
We could use   "{f[2] or ''}" I guess

Reply:
Absolutely. More elegant that way. Changed.